hypertrace-graphql-label-application-rules-schema/src/main/java/org/hypertrace/graphql/label/application/rules/dao/LabelApplicationRuleResponseConverter.java

```diff
@@ -74,7 +74,12 @@
     }
     return action.map(
         labelAction ->
-            new ConvertedLabelApplicationRuleData(data.getName(), conditionList, labelAction));
+            new ConvertedLabelApplicationRuleData(
+                data.getName(),
+                conditionList,
+                labelAction,
+                data.getEnabled(),
```

Comment:
Did we handle backwards compat at the config layer? Or planning to handle it manually since this full feature isn't exposed yet? Current impl will default any pre-existing rules to disabled since that's the default value of the new boolean.

Reply:
There are no existing label application rule. Checked in saas-dev.